documents/postman/staging-simplified.postman_environment.json

```diff
@@ -0,0 +1,71 @@
+{
+	"id": "1acffa98-51c5-4ba7-87fc-9f3d460c38ba",
+	"name": "development Copy",
+	"values": [
+		{
+			"key": "email-from",
+			"value": "solutions",
+			"enabled": true
+		},
+		{
+			"key": "email-template-id",
+			"value": "",
+			"enabled": true
+		},
+		{
+			"key": "notification-api-url",
+			"value": "https://staging-api.va.gov/vanotify",
+			"enabled": true
+		},
+		{
+			"key": "notification-id",
+			"value": "",
+			"enabled": true
+		},
+		{
+			"key": "service-api-key",
+			"value": "",
+			"enabled": true
+		},
+		{
+			"key": "service-id",
+			"value": "",
+			"enabled": true
+		},
+		{
+			"key": "service-name",
+			"value": "VA Notify",
+			"enabled": true
+		},
+		{
+			"key": "sms-sender-id",
+			"value": "",
+			"enabled": true
+		},
+		{
+			"key": "sms-template-id",
```

Comment:
Is this variable needed/used here? If so - for which endpoint in this simplified collection?